shell/src/main/java/alluxio/cli/GetConfKey.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.cli;
+
+import alluxio.PropertyKey;
+
+import com.google.common.collect.ImmutableMap;
+
+import java.util.Map;
+
+/**
+ * Utility for printing Alluxio configuration key.
+ */
+public final class GetConfKey {
+  private static final String USAGE = "USAGE: GetConfKey [ENV_VARIABLE_NAME]\n\n"
+      + "GetConfKey prints the configuration key given an environment variable name. "
+      + "If the key is invalid, the exit code will be nonzero.";
+
+  private static final Map<String, String> ENV_VIOLATORS = ImmutableMap.of(
+      "ALLUXIO_UNDERFS_S3A_INHERIT_ACL", "alluxio.underfs.s3a.inherit_acl",
+      "ALLUXIO_MASTER_FORMAT_FILE_PREFIX", "alluxio.master.format.file_prefix",
+      "AWS_ACCESSKEYID", "aws.accessKeyId",
+      "AWS_SECRETKEY", "aws.secretKey"
+  );
+
+  /**
+   * Prints the help message.
+   *
+   * @param message message before standard usage information
+   */
+  public static void printHelp(String message) {
+    System.err.println(message);
+    System.out.println(USAGE);
+  }
+
+  /**
+   * Implements get configuration key.
+   *
+   * @param args list of arguments
+   * @return 0 on success, 1 on failures
+   */
+  public static int getConfKey(String... args) {
+    switch (args.length) {
+      case 0:
+        printHelp("Missing argument.");
+        return 1;
```

Comment:
nit: do different non-zero error codes make sense?. ignore if a trivial mapping does not exist.

Reply:
I don't think it matters. We only differentiate zero with non-zero return code.